docker/pypi/wmagent/bin/manage

```diff
@@ -165,58 +204,61 @@ status_of_agent(){
 clean_agent(){
     stop_agent;
     if  _init_valid $wmaInitActive ; then
-        echo "Cleaning WMAgent..."
+        echo "$FUNCNAME: Cleaning WMAgent..."
         rm -rf $WMA_INSTALL_DIR/*
         rm -f $WMA_CONFIG_DIR/config.py;
 
         case $AGENT_FLAVOR in
             'mysql')
-                _exec_mysql "drop database $wmaDBName"
-                _exec_mysql "create database $wmaDBName"
-                rm -f $wmaInitAgent
-                rm -f $wmaInitSqlDB
+                clean_mysql
                 ;;
             'oracle')
-                echo "$FUNCNAME: Not Implemented"
+                clean_oracle
                 ;;
             *)
                 echo "$FUNCNAME: ERROR: Unknown or not set Agent Flavor"
                 return $(false)
                 ;;
         esac
-
+        rm -f $wmaInitAgent
     else
-        echo "ERROR: This agent is not yet activated. Cannot clean it."
+        echo "$FUNCNAME: ERROR: This agent is not yet activated. Cannot clean it."
         return $(false)
     fi
 }
 
 # Wipe out MySQL and Oracle databases
 # Will cause next start to recreate databases
 clean_mysql(){
-    [[ -z $MYSQL_USER ]] && { echo "Not using MySQL..."; exit 1 ; }
-    echo "Dropping MySQL DB... ";
-    _exec_mysql "drop database $wmaDBName"
-    rm -f $wmaInitSqlDB
+    [[ -z $MYSQL_USER ]] && { echo "$FUNCNAME: ERROR: Not using MySQL..."; exit 1 ; }
+    local errVal=0
+    echo "$FUNCNAME: Dropping MySQL DB... ";
+    _exec_mysql "drop database $wmaDBName"   || let errVal+=$?
+    _exec_mysql "create database $wmaDBName" || let errVal+=$?
```

Comment:
But if you drop the database, why would you want to create any tables? I see database and table creation tightly coupled, meaning that whenever tables need to be created, the database needs to either be created or verified.

Reply:
Maybe a different question is, is this "create database.." query fixing any problems that you had already spotted? Unless I missed it, it seems to be a new addition in this PR.

Reply:
Ok, I tested it few times without the recreation lines. here is the result
the only difference is that  if the agent is not initialized yet we see WARNING of the sort:
```
_sql_db_isclean: WARNING: Database wmagent does not exist. 'manage init-agent' should be executed before starting the agent.
```

and 
```
(WMAgent-2.3.0) [cmst1@vocms0290:current]$ manage db-prompt
ERROR 1049 (42000): Unknown database 'wmagent'
```

The one thrown during the initialization process, I have turned earlier into a `WARNING` instead of a hard error. So we must be fine.